target/linux/ramips/dts/rt3352.dtsi

```diff
@@ -324,8 +324,8 @@
 
 		clocks = <&sysc 12>;
 
-		resets = <&sysc 21>;
-		reset-names = "fe";
+		resets = <&sysc 21>, <&sysc 23>;
+		reset-names = "fe", "esw";
```

Comment:
Please apply these changes to RT5350, too. This is the vendor driver behavior. Ref:
https://github.com/hanwckf/rt-n56u/blob/23387b278a7cf728748af606760758f5d59d1451/trunk/linux-3.4.x/drivers/net/raeth/ra_eth.c#L24

Reply:
> Makes sense. By any chance, do you have a better device to test that, than MPR-A2 which is single port and barely even works because of 32MB RAM? 

Unfortunately, I don't have such a device. The Ralink RT series is too old. Most of them have small RAM.

> Also, RT3050 is probably affected too - 

Agree. I forgot this SoC, RT305{0,2}.

> on the other hand I could connect to that MPR-A2 even without changes to its device tree.

Perhaps only some special devices can trap this problem. I have two MT7628 routers that have always been stable in the past few years.

Reply:
Okay, should be fixed now.